### PR TITLE
Update Czech translation to v3.3.0

### PR DESCRIPTION
**Pull request by panther7**

### CODE REVIEW

File: elevenclock/lang/lang_cs.py [hunk 1]
```diff
@@ -11,21 +11,21 @@
 # Meke special attention to elements like ":", etc.
 
 lang_3_3 = {
-    "Custom format rules:": "",
+    "Custom format rules:": "Pravidla vlastního formátu:",
     "Any text can be placed here. To place items such as date and time, please use the 1989 C standard. More info on the following link": "",
     "Python Date and time values": "",
     "To disable the zero-padding effect, add a # in bethwwn the % and the code: non-zero-padded hours would be %#H, and zero-padded hours would be %H": "", # Here please don't modify the %H and %#H values
```
Comment (panther7) [inline elevenclock/lang/lang_cs.py:17]: "bethwwn"? :)

@martinet101

Comment (martinet101) [reply elevenclock/lang/lang_cs.py:17]: Yes, it was between, it has been fixed in the English translation, but I have to replace this on the entire code.

**Review by martinet101 — APPROVED** [review]

**Comment by martinet101** [comment]
@panther7, is this ready? It is to merge it and publish the update, because if I don't publish it today I'll need to wait until next weekend...

**Comment by panther7** [comment]
Tonight

**Comment by panther7** [comment]
Is possible add breaking lines for long text?

**Comment by martinet101** [comment]
I just need to add text wrapping

**Review by martinet101 — APPROVED** [review]

**Pull request merged**